jax/interpreters/pxla.py

```diff
@@ -743,9 +742,10 @@ def xla_pmap_impl(fun: lu.WrappedFun, *args, backend, axis_name, axis_size,
                           ("fingerprint", fingerprint))
   return compiled_fun(*args)
 
+
 @lu.cache
 def parallel_callable(fun: lu.WrappedFun,
-                      backend_name: Optional[str],
+                      backend: Optional[str],
```

Comment:
`backend_name` is a slightly better name IMO, because `backend` sounds more like a real backend object from the XLA client.